CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### New features
 
 * [#4650](https://github.com/bbatsov/rubocop/issues/4650): Add new `Style/StringHashKeys` cop. ([@donjar][])
+* [#1583](https://github.com/bbatsov/rubocop/issues/1583): Add a quiet formatter. ([@drenmi][])
+* [#4794](https://github.com/bbatsov/rubocop/issues/4794): Fix an error in `Layout/MultilineOperationIndentation` when an operation spans multiple lines and contains a ternary expression. ([@rrosenblum][])
+* [#4909](https://github.com/bbatsov/rubocop/issues/4909): Make `Rails/HasManyOrHasOneDependent` aware of multiple associations in `with_options`. ([@koic][])
```

Comment:
Thank you for fixing the CHANGELOG.md. It seems that it was added to a position different from intention by `merge=union` of .gitattributes.

Only one point.
I intended to add `#4909` to `Bug fixes`. Would you please change from `New features` to `Bug fixes`? Also, perhaps `#4794` seems to be `Bug fixes`.

Reply:
Yeah, that makes sense.

Reply:
Thank you! fixed.